modular_zubbers/modules/code/alt_job_titles.dm

```diff
@@ -0,0 +1,228 @@
+/**
+ * This is the file you should use to add alternate titles for each job, just
+ * follow the way they're done here, it's easy enough and shouldn't take any
+ * time at all to add more or add some for a job that doesn't have any.
+ */
+
+/datum/job/ai
+	alt_titles = list(
+		"AI",
+		"Station Intelligence",
+		"Automated Overseer",
+		"Language Model",
+		"Mind Scan",
+		"Remote Helper Agent",
+	)
+
+/datum/job/assistant
+	alt_titles = list(
+		"Assistant",
+		"Civilian",
+		"Tourist",
+		"Businessman",
+		"Businesswoman",
+		"Trader",
+		"Entertainer",
+		"Freelancer",
+		"Artist",
+		"Off-Duty Staff",
+		"Off-Duty Crew",
+		"Collector",
+		"Maintenance Dweller",
+		"Hobbyist",
+		"Intern",
```

Comment:
```suggestion

```
we already have intern automatically applied to players with less than 50 hours onto the server, while as funny as it may be, I wouldn't like **John McGrey (Intern Intern)**